pkg/apis/config/defaults.go

```diff
@@ -85,6 +85,23 @@ type Defaults struct {
 type ClassAndBrokerConfig struct {
 	BrokerClass   string `json:"brokerClass,omitempty"`
 	*BrokerConfig `json:",inline"`
+	// BrokerClassSpec set the multiple configurations of different brokerClass.
+	// Different brokerClass use corresponding brokerConfig for all the namespaces that
+	// are not in NamespaceDefaultBrokerConfigs.
```

Comment:
why are we leaving broker-class based defaults for namespace defaults out?

Reply:
oh no, we are not leaving, the annotation description is wrong